zips/draft-ecc-zbloc.md

```diff
@@ -0,0 +1,268 @@
+
+* a "Governance Proposal", expressing a proposed change in Zcash Governance;
+* a "Funding Proposal", expressing a proposed disbursement of Protocol-defined Ecosystem Funding.
+
+Proposals have a Deadline (expressed as a Deadline Height if [^draft-ecc-onchain-accountable-voting] is used). If the Deadline is reached, they are implicitly Rejected. This does not necessarily preclude reintroducing the same or a similar Proposal.
+
+### Governance Proposals
+
+At least the following changes to Zcash Governance MUST be proposed and approved by a Governance Proposal before they can take effect:
+
+* Changes to Zcash consensus rules. In this case Approval is REQUIRED in order for an activation height to be set.
+* Substantive changes to the ZIP Process defined in ZIP 0 [^zip-0000].
+* Substantive changes to this ZIP defining the zBloc.
+* Changes to the set of zBloc Constituencies and/or the distribution of Voting Units among those Constituencies.
+* Changes to the policy a Constituency uses for its decision-making.
+
+The ZIP Editors will modify ZIP 0 to define other categories of change to Zcash Governance that MUST or SHOULD require Approval of a Governance Proposal.
+
+Unrelated changes SHOULD NOT be bundled into a single Governance Proposal. This does not prevent a single Governance Proposal from being used to decide whether to adopt a proposed Network Upgrade involving multiple consensus and/or process changes, since those are considered related. During the process of drafting Governance Proposals, the ZIP Editors MAY request that proposals be split in order to satisfy this rule, or merged if they are sufficiently interdependent.
+
+### Disputes
+
+If any zBloc Constituency, or the ZIP Editors, have concerns relating to the conduct of any vote, they MAY raise a dispute. Possible reasons to raise a dispute include believing that:
+
+* A Constituency or any holders of keys for its Voting Units were unduly pressured or coerced.
+* A Constituency voted contrary to its intended position (due to key compromise or error, for instance), or contrary to its stated decision-making policy.
+* A Constituency failed to disclose a relevant conflict of interest.
+* There were technical irregularities in the use of the voting mechanism.
+
+If a dispute is raised, the Constituencies SHALL first resolve any technical issues as far as possible (e.g. by rotating keys), and then make one or more Governance Proposals intended to resolve the situation to the satisfaction of the community.
+
+### Exceptional decisions to remediate security vulnerabilities
+
+The governance processes specified in this ZIP are not intended to prevent consensus node implementors from making consensus changes that they reasonably believe are critical to maintain the security of the Zcash Mainnet network and/or Zcash users. Such changes might need to be developed and released according to a controlled disclosure process that precludes public voting.
```

Comment:
This is also useful to copy to the community-and-coinholder proposal, or to factor out into a shared preamble.

Reply:
Again, community-and-coinholder only handles grants, not any other aspect of governance.